rs-matter/src/dm/clusters/noc.rs

```diff
@@ -624,4 +631,21 @@ impl ClusterHandler for NocHandler {
                 .add_trusted_root_cert(sess.get_session_mode(), request.root_ca_certificate()?.0)
         })
     }
+
+    fn handle_set_vid_verification_statement(
+        &self,
+        _ctx: impl InvokeContext,
+        _request: SetVIDVerificationStatementRequest<'_>,
+    ) -> Result<(), Error> {
+        Ok(()) // TODO
+    }
+
+    fn handle_sign_vid_verification_request<P: TLVBuilderParent>(
+        &self,
+        _ctx: impl InvokeContext,
+        _request: SignVIDVerificationRequestRequest<'_>,
+        _response: SignVIDVerificationResponseBuilder<P>,
+    ) -> Result<P, Error> {
+        todo!()
```

Comment:
Ditto here: either `unimplemented!` (if we are sure this guy should never be called) or else we need to implement it?